### ISSUE TITLE
Submission protos for camera-only detection?

### ISSUE THREAD

**Issue by zhiqi-li** [waymo-research/waymo-open-dataset#447]
I am wondering which submission proto should be used for camera-only detection?

**Comment by joonjeon** [comment]
Hi!

Same question here. I suppose the Waymo Challenge committee should post detailed instructions (and/or protobuf definitions) on exporting submission proto binary for camera-only detection.

And besides this, speaking about camera-only detection, I am wondering when the updated pip-package for waymo-open-dataset will be available. The current version does not support `CAMERA_ONLY_DETECTION_3D` for task enumeration value. Though it is possible to build from the latest source repo, I would appreciate if they could release the updated pip-package as well.

**Comment by zoeyyang1** [comment]
Hi @joonjeon, the updated pip package is uploaded. Please install the latest pip 1.4.5. Thank you.

**Comment by joonjeon** [comment]
Hmm... that's strange.
I checked PyPI website but it shows up like this:
![image](https://user-images.githubusercontent.com/29110031/165252294-07a895c7-25f7-4275-ad08-3a0c8f1c3e98.png)

When will the 1.4.5 package be released?

**Comment by hfslyc** [comment]
Hi,

The 1.4.5 has been released. Please let us know if more there is any issue when creating the submission.

Best,
Wayne, on behalf of the Waymo Open Dataset team

**Comment by zhiqi-li** [comment]
> Hi,
> 
> The 1.4.5 has been released. Please let us know if more there is any issue when creating the submission.
> 
> Best, Wayne, on behalf of the Waymo Open Dataset team

I got this error https://github.com/waymo-research/waymo-open-dataset/issues/468#issue-1215018353 while performing the evaluation. Currently, I can't get the LET-AP of my model locally. But I already created the submission of the validation set and successfully got the reasonable LET-AP from the server.

**Comment by hfslyc** [comment]
@zhiqi-li Can you confirm that the error still exists in pip package 1.4.5?

**Comment by Cc-Hy** [comment]
> > Hi,
> > The 1.4.5 has been released. Please let us know if more there is any issue when creating the submission.
> > Best, Wayne, on behalf of the Waymo Open Dataset team
> 
> I got this error [#468 (comment)](https://github.com/waymo-research/waymo-open-dataset/issues/468#issue-1215018353) while performing the evaluation. Currently, I can't get the LET-AP of my model locally. But I already created the submission of the validation set and successfully got the reasonable LET-AP from the server.

@zhiqi-li 
Hello, can you tell me how you create the submission  that you upload to the online server?
Thank you very much.

**Comment by zhiqi-li** [comment]
> @zhiqi-li Can you confirm that the error still exists in pip package 1.4.5?

I do confirm this error exists and only exists in 1.4.5. 
```
Traceback (most recent call last):
  File "projects/mmdet3d_plugin/datasets/eval_waymo.py", line 694, in <module>
    eval(pd_file, gt_file)
  File "projects/mmdet3d_plugin/datasets/eval_waymo.py", line 240, in eval
    metrics = compute_ap(decoded_outputs)
  File "projects/mmdet3d_plugin/datasets/eval_waymo.py", line 355, in compute_ap
    gt_bbox, gt_type, gt_frame_id, difficulty)
  File "projects/mmdet3d_plugin/datasets/eval_waymo.py", line 383, in build_waymo_metric
    config=waymo_metric_config.SerializeToString())
  File "/home/xxx/miniconda3/envs/py3.7/lib/python3.7/site-packages/waymo_open_dataset/metrics/ops/py_metrics_ops.py", line 43, in detection_metrics
    return metrics_module.detection_metrics(
AttributeError: module '012ff3e36e3c24aefc4a3a7b68a03fedd1e7a7e1' has no attribute 'detection_metrics'

```

**Comment by hfslyc** [comment]
Hi,

This issue should be addressed now with pip 1.4.6. Please let us know if there is more issue.

Thanks,
Wayne, on behalf of the Waymo Open Dataset team.

**Comment by zhiqi-li** [comment]
> Hi,
> 
> This issue should be addressed now with pip 1.4.6. Please let us know if there is more issue.
> 
> Thanks, Wayne, on behalf of the Waymo Open Dataset team.

Hi,  everything goes on well with 1.4.6.  Thanks for your efforts.
